src/map/packets_struct.hpp

```diff
@@ -4309,6 +4309,24 @@ struct PACKET_ZC_TAKEOFF_EQUIP_ALL_ACK {
 DEFINE_PACKET_HEADER(ZC_TAKEOFF_EQUIP_ALL_ACK, 0x0bae);
 #endif  // PACKETVER_MAIN_NUM >= 20210818 || PACKETVER_RE_NUM >= 20211103
 
+#if PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200724
```

Comment:
Please dont add something to Hercules' packets_struct.hpp file.
If it is not in this file add it to packets.hpp